src/scenes/home/faq/questions.json

```diff
@@ -34,7 +34,7 @@
     },
     {
       "question": "How can I help, if I can't afford to donate to Operation Code?",
-      "answer": "In addition to requiring financial support, we also need volunteers and interns. The larger our community, the more we can spread the word about our work. Also, remember that every <a href=\"/donate\">donation</a>, no matter how modest, brings us closer to our goals."
+      "answer": "In addition to requiring financial support, we also need <a href=\"http://op.co.de/volunteer\" target=\"_blank\">volunteers</a> and interns. The larger our community, the more we can spread the word about our work. Also, remember that every <a href=\"/donate\">donation</a>, no matter how modest, brings us closer to our goals."
```

Comment:
It's our fault for not having Storybook properly setup, but will you please utilize the ExternalLink component instead of an anchor tag?

Reply:
@kylemh We might have issues with using `OutboundLink` inside of a  JSON file such as this -  It would probably require wiring up something to ensure that any component that uses this JSON content has a reference to the `OutboundLink` component so the correct HTML is eventually rendered, introduced more coupling.

There's also several other places in this file where hardcoded anchor tags are already used - it seems that's the convention so far.

Reply:
I see that now... Not a big fan of dangerouslySetInnerHTML, but it looks like that's convention.